afdko/Tools/SharedData/FDKScripts/BezTools.py

```diff
@@ -1,9 +1,8 @@
-from __future__ import print_function, absolute_import
```

Comment:
the `absolute_import` should stay. It simply makes the import machinery in python2 behave like python3 (e.g. disallowing implicit relative imports)